osquery/config/config.cpp

```diff
@@ -208,6 +179,51 @@ class Schedule : private boost::noncopyable {
   friend class Config;
 };
 
+bool Schedule::Step::operator()(PackRef& pack) {
+  return pack->shouldPackExecute();
+}
+
+void Schedule::add(PackRef&& pack) {
+  remove(pack->getName(), pack->getSource());
+  packs_.push_back(pack);
```

Comment:
`<  packs_.push_back(pack);`
`>  packs_.push_back(std::move(pack));`

Reply:
This should happen automatically, `Pack` does not have a copy constructor.

Reply:
It does not happen automatically besides identifier pack is of type std::shared_ptr<Pack> not Pack.

Reply:
Anyway since it is std::shared_ptr so overhead is not that much.